lib/beaker/dsl/install_utils.rb

```diff
@@ -1453,6 +1453,104 @@ def build_ignore_list(opts = {})
         ignore_list << '..' unless ignore_list.include? '..'
         ignore_list
       end
+
+      # Loop through all windows hosts and install the correct certs to fix
+      # PUP-2365 which allows the proper download of modules from the forge
+      #
+      # @return nil
+      # @api dsl
+      def install_forge_cert
+        hosts.each do |host|
+          if host['platform'] =~ /windows/
+            configure_puppet_on(host,opts)
+          end
+        end
+      end
+
+      # This method is a fix for PUP-2365 when a windows agent cannot download
+      # modules from the forge because it doesn't have the appropriate
+      # certificates installed to validate the https connection.
+      #
+      # This method installs the certifcates required in order to that functionality
+      # to work as expected.
+      #
+      # @param [Host] host             A host object
+      #
+      def install_forge_cert_on(host)
```

Comment:
Talked this over.  We think that the way to go would be to have a generic 'install_cert_on' and then match it with a pre-suite test that installs the correct cert on the hosts that you want affected.  Otherwise this method is just too tightly constrained.